modules/postgres/metadata.json

```
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://www.postgresql.org/",
+    "maintainers": [
+        {
+            "email": "177060987+freeform-andre@users.noreply.github.com",
+            "github": "freeform-andre",
+            "name": "freeform-andre"
+        }
+    ],
+    "repository": [
+        "github:postgres/postgres"
```

Because the `repository` field has only one entry `github:postgres/postgres` the URL in source.json has to come from the postgres github repo (refer back to the error message you posted). To allow `ftp.postgresql.org` URLs as well, you can add a new entry here (`"https://ftp.postgresql.org/"`).

---

I have added this and unfortunately the issue persists 😞 
```suggestion
        "https://ftp.postgresql.org/",
        "github:postgres/postgres"
```

---

Fixing the check in https://github.com/bazelbuild/bazel-central-registry/pull/3933

---

Thanks!

---

This should be fixed now, can you rebase the PR?

---

Rebased and it works now!